### PR TITLE
Add Sentry, FOSS error monitoring tool

### PR DESCRIPTION
**Pull request by bruno-garcia**
Adding link to [Sentry](https://github.com/getsentry/sentry) integration with ASP.NET Core and `Microsoft.Extensions.Logging`.

### CODE REVIEW

**Comment by guardrex** [comment]
@pranavkm, Sentry has been around for a long time, and their ASP.NET Core provider just hit release (1.0). Are you (others?) good with surfacing it along with the mature providers suggested?

File: aspnetcore/fundamentals/logging/index.md [hunk 1]
```diff
@@ -753,6 +753,7 @@ Third-party logging frameworks that work with ASP.NET Core:
 * [KissLog.net](https://kisslog.net/) ([GitHub repo](https://github.com/catalingavan/KissLog-net))
 * [Loggr](http://loggr.net/) ([GitHub repo](https://github.com/imobile3/Loggr.Extensions.Logging))
 * [NLog](http://nlog-project.org/) ([GitHub repo](https://github.com/NLog/NLog.Extensions.Logging))
+* [Sentry](http://sentry.io/) ([GitHub repo](https://github.com/getsentry/sentry-dotnet))
```
Comment (guardrex) [inline aspnetcore/fundamentals/logging/index.md]: Use the secure, redirected endpoint: `https://sentry.io/welcome/`

**Comment by bruno-garcia** [comment]
I'll try to make my case:

The Raven SDK has been around for many years and has over 400k downloads on NuGet. It's been used with ASP. NET Core too for a while now.
The new SDK had 10k downloads of its preview bits. There are hundreds of thousands of events coming into the SaaS (sentry.io) every week with the SDK linked. Sentry itself has great support for dotnet events.

**Comment by guardrex** [comment]
@bruno-garcia Sure ... I suspect they'll green-light it, but we just need to make sure. They might have some kind of *maturity test* that they'd like to apply.

WRT the IIS Modules entry, I think on a future PR that I'm going to cross-link there to the list in the Logging topic. Don't sweat it right now. Let's just leave what you did. I need to open and issue and address a couple of things at the same time.

File: aspnetcore/fundamentals/logging/index.md (hunk 1 above)
Comment (bruno-garcia) [reply aspnetcore/fundamentals/logging/index.md]: Oh my bad. Typed in and forgot the 's'.

**Comment by pranavkm** [comment]
> Are you (others?) good with surfacing it along with the mature providers suggested?

Question's better suited for @Eilon \ @davidfowl \ @DamianEdwards

**Comment by guardrex** [comment]
... and @danroth27.

For the record, the current list of recommended providers is:

* elmah.io
* JSNLog
* KissLog.net
* Loggr
* NLog
* Serilog

&plus; Sentry if approved on this PR.

**Comment by Eilon** [comment]
For my team there's no "maturity test" - I just check that the links work and seem to do what they claim to do. It's not even a list of *recommended* providers at https://github.com/aspnet/Logging/blob/master/README.md, just a list of providers that *exist*.

**Comment by guardrex** [comment]
> list of providers that exist

WRT the topic, "recommended" might be the way that readers inadvertently take it. Even I didn't know, which is why I asked. I'll ask Rick on the other issue what he wants to do, if anything.

Thanks everyone!

**Pull request merged**